programs/programs/nc/nc_head_start/tests.py

```
@@ -0,0 +1,573 @@
+        # Should be ELIGIBLE due to housing cost adjustment
+        self.assertTrue(
+            eligibility.eligible, "Should be ELIGIBLE - housing cost adjustment brings income below 130% FPL"
+        )
+
+        # Should have 1 eligible child (the 3-year-old) - parent doesn't count since not pregnant
+        eligible_children = [m for m in eligibility.eligible_members if m.member.relationship == "child" and m.eligible]
+        self.assertEqual(len(eligible_children), 1, "Should have 1 eligible child (3-year-old)")
+
+        # Value should be preschool rate * 12
+        # Durham County preschool rate: $1,167/month (1000 in MARKET_RATES_DATA)
+        self.assertEqual(eligibility.value, 1167 * 12, "Estimated value should be $14,004 (preschool rate × 12)")
```

Ruff (0.14.8)

String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)